### ISSUE TITLE
Make alt text on alumni images more accessible

### ISSUE THREAD

**Issue by bbsmooth** [freeCodeCamp/freeCodeCamp#51033]
### Describe the Issue

This is a minor issue, but I think it needs to be addressed. The current alt text for the three alumni images on the  main landing page is just the person's name. This is merely duplicating information that is already provided and doesn't give people who can't see the image the same information as people who can see the image.

### Affected Page

https://www.freecodecamp.org/

### Steps to Reproduce

1. Go to 'https://www.freecodecamp.org/'
2. View alt text for each alumni image with the browser's dev tools inspector

### Expected behavior

Alt text for each image should be updated to be a description of the image. This doesn't need to be extremely detailed. Just hit the basic details. Perhaps we could reach out to each of these people and ask them to write their own alt text?

### System

N/A

**Comment by dhruvchandak30** [comment]
@bbsmooth  Can I work in this issue?

**Comment by dhruvchandak30** [comment]
```

 alt="Shawn Wang"
```
To
```
 alt="Alumni: Shawn Wang @Amazon"
```
 
 Will this work?

**Comment by mmatsumoto1026** [comment]
It seems that to make `alt` text of alumni images more accessible, we should decide what (if any) information or function the images have.

Particularly, how about simply stating the information conveyed from the expressions and atmosphere of each alumni as follows.

- Shawn Wang is smiling lively at the top of a tall mountain.

- Sarah Chima is sitting in a chair and smiling with poise.

- Emma Bostian is in the garden with bright smile.

I've refered the [Tips and Tricks in Images Tutorial document of W3C](https://www.w3.org/WAI/tutorials/images/tips/) and `alt` text of images in [the university of Alabama HP](https://www.ua.edu/).

**Comment by dhruvchandak30** [comment]
@mmatsumoto1026  @bbsmooth  I have made the PR. Can you please review it.
#51042

**Comment by bbsmooth** [comment]
@dhruvchandak30 I'm not sure we have settled on the details that should be included in the alt text. Your PR may be a little premature.

> It seems that to make alt text of alumni images more accessible, we should decide what (if any) information or function the images have.

Agreed. The best alt text describes the information that is being conveyed through the image. That can vary depending on the context the image is being used in. I'm curious to know what information you all think these images convey? Perhaps someone who was involved in the selection of these three people can give us some insight?

Personally, I think one of the messages these images are conveying is that fCC strives to be an inclusive environment by showing that people other than traditional white guys can succeed in this industry. So I think it might be important to describe what these people look like so people who can't see the images also get this information. Here's an article that explains it a little better than I can:

[The case for describing race in alternative text attributes](https://ux.shopify.com/the-case-for-describing-race-in-alternative-text-attributes-a093380634f2)

**Comment by dhruvchandak30** [comment]
So should I also mention 
Color of skin, Hairstyle, What are they wearing in addition to the Background and type of Smile?

**Comment by bbsmooth** [comment]
I think you should just wait until more people have had a chance to give their input. This isn't something that needs to happen immediately.

**Comment by alivarastepour** [comment]
I agree with @bbsmooth about the whole race thing; additionally, I think the same goes for their gender too.

**Comment by DylanDevelops** [comment]
> [The case for describing race in alternative text attributes](https://ux.shopify.com/the-case-for-describing-race-in-alternative-text-attributes-a093380634f2)

This article is very interesting. Thank you for sharing! It's things like this that I forget when it doesn't impact me but incredibly important for someone who has a disability.

**Comment by DylanDevelops** [comment]
We could do something like this:

Shawn Wang:
> Shawn Wang, a software engineer living in Singapore and working at Amazon, standing at the top of a tall mountain.

Sarah Chima:
> Sarah Chima, a software engineer living in Nigeria and working at ChatDesk, sitting at their desk.

Emma Bostian
> Emma Bostian, a software engineer living in Sweden and working at Spotify, posing for a photo in nature.

Let me know if these work. I'm still unsure how we should implement race. Please leave your feedback!

**Comment by raisedadead** [comment]
Thanks Dylan! These are perfect, 

@dhruvchandak30, You can go ahead and use the following like @DylanDevelops recommends:

Image ofStudents at Laptop:
> Five freeCodeCamp students at a local study group in South Korea working together on a laptop.

Shawn Wang:
> Shawn Wang, a software engineer living in Singapore and working at Amazon, standing at the top of a tall mountain.

Sarah Chima:
> Sarah Chima, a software engineer living in Nigeria and working at ChatDesk, sitting at their desk.

Emma Bostian
> Emma Bostian, a software engineer living in Sweden and working at Spotify, posing for a photo in nature.

Let us know if you need further help.

**Comment by bbsmooth** [comment]
I'm going to push back a little on these.

> Shawn Wang, a software engineer living in Singapore and working at Amazon, standing at the top of a tall mountain.

The image itself does not show any indication that Shawn lives in Singapore or works for Amazon. This is all information provided in the content next to the image, not the image itself.

> Sarah Chima, a software engineer living in Nigeria and working at ChatDesk, sitting at their desk.

Again, the image does not show any indication that Sarah is a software engineer, or lives in Nigeria, or works at ChatDesk. 

> Emma Bostian, a software engineer living in Sweden and working at Spotify, posing for a photo in nature.

I think you know what I'm going to say here. The image does not show any indication that Emma is a software engineer, or lives in Sweden, or works at Spotify.

The alt text should only convey information that can be gleaned from the image. Sighted people get the information above from the text next to the image, as will non-sighted people, so it doesn't need to be repeated in the alt text.

If there is a reason we don't want to include race/ethnicity/gender in these alt descriptions then so be it. But really, that's the only additional information these images are conveying.

**Comment by raisedadead** [comment]
Hi @bbsmooth,

We have gotten into bikeshedding territory here. So to cut to the chase, the expected behavior:

> Alt text for each image should be updated to be a description of the image. This doesn't need to be extremely detailed. Just hit the basic details. Perhaps we could reach out to each of these people and ask them to write their own alt text?

We want to avoid reaching out to the people who have generously provided us with these testimonials to write down an alt text years after we asked them for the testimonials :)

Shawn, who I know closely, has worked with multiple different orgs since then and has started/funded multiple startups.

So, if we want the alt-text to be improved with details, we have to add that ourselves with the information we have at the time of the testimonial.

I think we should stick to the person's name, maybe update it to something like "Shawn's profile picture," and that's it. 

I am OK with a piece of more descriptive information, but it has come from the information we have had at the time - which again is already available to both sighted and non-sighted people - there is no new information to be discovered or added.   

I'll defer to your final thoughts before we wind this up :)

**Comment by raisedadead** [comment]
> I think these images help convey a message of inclusiveness

Oh - yes, we are on the same page here. That's what we want; the goal is fine.

I am only thinking objectively as a maintainer here to keep moving the work. I fear discussions become stale if we "wait" too long. So when I was deferring to you, I was asking: 

"Let's do it. Give us your version of the code changes instead of waiting for more feedback" - you, as a contributor own very much what the message should be!

I am only trying to make sure we know the constraints that we have around the change - I think I laid it out in my previous comment. Hope it clears up any confusion! :)

**Comment by DylanDevelops** [comment]
Are we going along with my changes or still deciding?

**Comment by raisedadead** [comment]
Hi @DylanDevelops - I think for now we agree that we will be duplicating the information we have (with the strings you proposed). So we should not do that.

At the very least, we should update it to "<Person Name>'s profile picture" to make it more verbose and useful.

**Comment by raisedadead** [comment]
I also want to request @bbsmooth to give us a few sample alt-text strings which convey inclusive information about the people.

Bruce, can you help us here since you are the best person to guide us? Thanks a lot.

**Comment by mmatsumoto1026** [comment]
After reading the comments and suggestions in this issue, my another suggestions for the alt text of each alumni image are shown in below.

- An Asian adult man, Shawn Wang's profile picture: He is standing on the high place, with a very tall mountain in the background.

- A Black lady, Sarah Chima's profile picture: She is sitting on a chair in a frugal room.

- A White woman, Emma Bostian's profile picture: She is posing for a photo in nature.

So far, these are what information I all think these images convey.

> Personally, I think one of the messages these images are conveying is that fCC strives to be an inclusive environment by showing that people other than traditional white guys can succeed in this industry.

I think so too, and In addition to that, I think fCC are trying to convey that people of all ages(they did not use the images of boys and girls), [gender](https://en.wikipedia.org/wiki/Women_in_Sweden), races, places of residence, and [living environments](https://en.wikipedia.org/wiki/Women_in_Nigeria) can participate the curriculum and learn anytime, anywhere. This should be the "normal" impact our alt text has on camper’s perception. So I've tried implementing the "normal" in alt text with using words like 'adult', 'frugal room', 'lady' and 'woman'.

But I'm not native English user, so if there is some advice for my suggestions, I think it will helps a lot.

**Comment by bbsmooth** [comment]
"Shawn Wang, a young-looking Asian man, smiling for a selfie with a snow-capped mountain in the background."

"Sarah Chima, a young-looking Black woman, smiling for the camera while sitting in a chair."

"Emma Bostain, a young-looking White woman, smiling for the camera in front of green foliage."

@mmatsumoto1026 I don't mean to take away from your contributions. I have been working on these over the past few days and you just happened to post yours before mine were done.

**Comment by mmatsumoto1026** [comment]
> I don't think mine are radically different from yours.

After checking our alt texts using google image search engine, it seems that your texts showed more images that are look like the ones in the landing page, so I think yours are not only more accessible but also more appropriate from an [SEO point of view](https://developers.google.com/search/docs/appearance/google-images?hl=en&visit_id=638268705244604930-6132769&rd=1#descriptive-alt-text). (To tell the truth, some of the texts I suggested ended up with [images that were not very similar](https://www.freepik.com/premium-photo/rear-young-asian-man-standing-with-looking-mountain-view-top-hill-national-park_14494505.htm) to the ones in landing page)

> The one thing we don't want to do is add information that isn't obvious in the images.

I certainly agree with you. Originaly, I used "frugal" because simply I couldn't find many furnitures in the Sarah's image, but now I find it difficult to think that it is obvious in the range that is reflected in the image.

Also, I agree with your use of the word "young-looking". Because I think it's important to start learning programming as early as possible (since there's so much to learn).

**Comment by DylanDevelops** [comment]
I just added a review to the active pull request. However, where would the user go to change the alt text of the study group in South Korea?

If we agree to update it, the new alt text should be:
> Five freeCodeCamp students at a local study group in South Korea working together on a laptop.

I assume this is the file to change:
https://github.com/freeCodeCamp/freeCodeCamp/blob/d23676ad8618a75a14412a2d4bdeebe11c077b68/client/src/components/landing/components/campers-image.tsx

The code says `alt={t('landing.hero-img-description')}`. I am unfamiliar with this syntax, so where does this point to?
